sync-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -81,12 +91,20 @@
     private volatile boolean replicationTerminated = false;
 
     public BasicPullStrategy(PullReplication pullReplication) {
-        this(pullReplication, null, null);
+        this(pullReplication,
+                null,
+                DEFAULT_CHANGES_LIMIT_PER_BATCH,
+                DEFAULT_MAX_BATCH_COUNTER_PER_RUN,
+                DEFAULT_INSERT_BATCH_SIZE,
+                DEFAULT_PULL_ATTACHMENTS_INLINE);
     }
 
     public BasicPullStrategy(PullReplication pullReplication,
                              ExecutorService executorService,
-                             PullConfiguration config) {
+                             int changeLimitPerBatch,
+                             int batchLimitPerRun,
+                             int insertBatchSize,
+                             boolean pullAttachmentsInline ) {
```

Comment:
Don't allow publicly changing these from the public interface, in this PR at least.